src/pscf/sweep/ParameterModifier.h

```diff
@@ -2,6 +2,7 @@
 #define PSCF_PARAMETER_MODIFIER_H
 
 #include <string>
+#include "util/containers/GArray.h"
 #include "util/containers/DArray.h"
 #include "pscf/sweep/ParameterType.h" // base class
```

Comment:
ParameterType isn't a base class of ParameterModifier, as the comment indicates

Also, includes should go from specific / higher level to generic / lower level later, e.g., includes of files from the pscf directory and namespace before util namespace, prdc after pscf, and rpc and rpg after prdc. This convention is designed to force developers to make sure that every header includes all the files it needs in order to be be parsed in isolation, without relying on prior inclusion of headers for related lower level types. This order makes it more likely that a violation of this convention will be caught because a header will be unparseable.